daemon/cmd/daemon_main.go

```diff
@@ -789,6 +789,9 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.Int(option.AuthMapEntriesName, option.AuthMapEntriesDefault, "Maximum number of entries in auth map")
 	option.BindEnv(vp, option.AuthMapEntriesName)
 
+	flags.StringSlice(option.ControllerGroupMetrics, []string{}, "List of controller group names for which to to enable metrics. Accepts 'all' and 'none'")
+	option.BindEnv(vp, option.ControllerGroupMetrics)
```

Comment:
Given the current modularization effort, I think it would be useful to move those to a separate `Config`, to allow registering the same flags in all components (agent, operator, clustermesh-apiserver, kvstoremesh). I guess that  the same could also apply for the metrics themselves: https://docs.cilium.io/en/latest/contributing/development/hive/#metric

Reply:
I added a cell.Config for the controller package. Working on possibly using a cell.Metrics now. It looks like this might involve a large refactor that would be outside the scope of this PR. If you have any suggestions based on my initial approach, let me know.

Reply:
The same approach worked for cell.Metrics! https://github.com/cilium/cilium/pull/26850/commits/d1b385be0a9113898bcf148f24aa285e3ba89c13#diff-ac54f170f197d1a1607b60527697e2c8228d0a3ca57046a7814f76393e6d7a50R64

Let me know if you have any qualms about this "package-scope shuffling" using Invoke.